content/rc/databases/high-availability.md

```diff
@@ -0,0 +1,29 @@
+Title: High Availability
+description:
+weight: 90
+alwaysopen: false
+categories: ["RC"]
+aliases: 
+
+---
+
+Database replication helps ensure high availability. When replication is enabled, your dataset is duplicated to create a replica, which is constantly synchronized with the primary dataset. Replication allows for automatic failover and greater fault tolerance, to help prevent data loss should something go wrong. 
```

Comment:
```suggestion
Database replication helps ensure high availability. When replication is enabled, your dataset is duplicated to create a replica, which stays in sync with the primary dataset. Replication allows for automatic failover and greater fault tolerance. This can prevent data loss in the event of a hardware, zone, or regional failure. 
```